tests/Run-UnitTests.ps1

```diff
@@ -1,39 +1,88 @@
 <#
 .SYNOPSIS
-    Runs Tests for Islandwood project.
+    Runs Functional Tests for Islandwood project.
```

Comment:
Can we just merge this with run-functionaltests?  Would be nice to only "maintain" 1 script.
Granted, once these are written, they usually don't have to be modified.